rspace-bench/src/test/scala/coop/rchain/rspace/RoundTripTest.scala

```diff
@@ -0,0 +1,35 @@
+package coop.rchain.rspace
+
+import bench.serialization._
+import coop.rchain.models._
+
+import org.scalatest._
+import org.scalatest.prop._
+import org.scalacheck._
+import scala.reflect.ClassTag
+
+class RoundTripTest extends FlatSpec with PropertyChecks with Matchers {
+  implicit override val generatorDrivenConfig =
+    PropertyCheckConfiguration(minSuccessful = 50, sizeRange = 250)
+
+  def roundTrip[A](in: A)(implicit s: Serialize2ByteBuffer[A]): A = {
+    val meta = s.encode(in)
+    val out  = s.decode(meta)
+    assert(out == in)
+    out
+  }
+
+  implicit val serialize = KryoSerializers.serializer(classOf[TaggedContinuation])
+
+  def roundTripSerialization[A: Serialize2ByteBuffer: Arbitrary](
+      implicit tag: ClassTag[A]
+  ): Unit =
+    it must s"work for ${tag.runtimeClass.getSimpleName}" in {
+      forAll { a: A =>
+        roundTrip(a)
+      }
+    }
+
+  import coop.rchain.models.testImplicits._
+  roundTripSerialization[TaggedContinuation]
```

Comment:
Please add tests for Par and Expr at least, and ideally for everything we have in RhoTypesTest.

After doing that, please also make sure that the tests pass even if you require `minSuccessful = 500` (should take ~1 minute per run), but don't commit that (it' just to make sure the build will be stable)